### DISCUSSION TITLE
Checklist for individual statuses, not just for publication

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-checklists#428] [Feature Requests]
One checklist improvement we can add is the ability to have checklists for different statuses.

For example, a "Pending" status may have simple requirements, but a "Published" status may have more complex requirements.

> I want to be able to use the pre-publish checklist that prevents them from changing the status to pending until the checklist is complete or something similar. Basically, our "pending" status is the author's last step before I or my editor go in and verify everything and publish it. I want to make it so that they cannot change it to pending (or equivalent status) until everything on the checklist is complete including custom rules such as our story formatting.

**Comment by stevejburge** [comment]
More on this from @jorr 

> I would suggest adding a UI choice or a filter that lets me choose
> the *status the user is prevented to set *when not having completed the
> Content Checklist. In our case, it would be much preferred to bar users
> from saving as Pending Review and leaving the Content Checklist to the
> editors to take care of. So it would be great to not be constrained only
> with 'publish' as forbidden status unless the content checklist is
> fullfilled. I wasn't able to find anything like that.

**Comment by stevejburge** [comment]
One more for this:

> Can we change from "before content is published" to "before content is saved as pending for review".
> I have a lot of contributors who need a checklist (requirements) before they send the content over to an editor for review.
> It would be helpful to have this as a feature of PublishPress.

**Comment by andergmartins** [comment]
👍

**Comment by stevejburge** [comment]
One more request for this today

**Comment by andergmartins** [comment]
Related to #100

**Comment by stevejburge** [comment]
This issue has a request for the Checklist to work after publishing https://github.com/publishpress/PublishPress-Checklists/issues/108

**Comment by stevejburge** [comment]
> I’d love to use this plugin, but as my workflow is for authors to submit posts for review, and an editor to then publish after reviewing, it is of no use.
> Given PublishPress’s main plugin is focused on workflows, it seems odd that this isn’t taken into account with this plugin.
> It is at the draft -> Pending status change that a checklist would be useful for me, and probably many other sites.

**Comment by stevejburge** [comment]
A request for Checklists for specific categories or tags

> The Checklists plugin does not let me create checklists for each category or post. It only allows you to set a universal checklist, which is not what I want.

**Comment by stevejburge** [comment]
Now shipped
